addons/contour/1.0.1/tmpl-kustomization.yaml

```diff
@@ -1,4 +1,7 @@
+namespace: $CONTOUR_NAMESPACE
+
 resources:
+- namespace.yaml
```

Comment:
I'd heard namespaces are like CRDs in that you can't create them with the same apply where they are used. Were you able to test this with the namespace not pre-existing and confirm it's ok?